include/ObjectManager.php

```diff
@@ -944,6 +958,63 @@ public function getResources(){
 	}
 
 ########################################## CREATE
+
+	/* LILAC - add kinship link */
+	public function addParentToHost($parentName, $childName, $exportConfiguration=FALSE){
+		
+		$error = "";
+		$success = "";
+		$code=0;
+		
+		try{
+
+			// Wants to add a parent 
+			$nhp = new NagiosHostPeer;
+			// Find host
+			$parentHost = $nhp->getByName($parentName);
+			if(!$parentHost) {
+				$code=1;
+				$error .= "Parent Host $parentName does not exists\n";
+			}
+
+			$childHost = $nhp->getByName($childName);
+			if(!$childHost) {
+				$code=1;
+				$error .= "Child Host $childName does not exists\n";
+			}
+
+			if($code==0){
+				$c = new Criteria();
+				$c->add(NagiosHostParentPeer::CHILD_HOST , $childHost->getId());
+				$c->add(NagiosHostParentPeer::PARENT_HOST, $parentHost->getId());
+				$parentRelationship = NagiosHostParentPeer::doSelectOne($c);
+				if($parentRelationship) {
+					$code=1;
+					$error .= "That parent relationship already exist.\n";
+				}else {
+					$tempParent = new NagiosHostParent();
+					$tempParent->setChildHost($childHost->getId());
+					$tempParent->setParentHost($parentHost->getId());
+					$tempParent->save();
+					$success .= "Parent added";
+				}
+
+				if( $exportConfiguration == TRUE )
```

Comment:
No need to test a value of a boolean: `if($exportConfiguration)`
is enough isn't it?